### ISSUE TITLE
Импорт моделей

### ISSUE THREAD

**Issue by Telhar** [BicycleStudio/Ewe#42]

**Comment by Swipes13** [comment]
Предлагаю начать помаленьку, взять определённый пакет, например, 3Dmax. Сделать скрипт (в силу того, что это проще) на экспорт простейших моделей. А дальше - развивать это в задачи.

**Comment by Swipes13** [comment]
Ты просто пустое сообщение оставил, я и добавил ))

**Comment by Telhar** [comment]
Согласен. Только, пожалуйста, не редактируй мои комментарии, а то мне аж не по себе стало, когда увидел))
